PhpAmqpLib/Wire/GenericContent.php

```diff
@@ -126,7 +122,7 @@ public function load_properties($r)
         while (true) {
             $flag_bits = $r->read_short();
             $flags[] = $flag_bits;
-            if (($flag_bits & 1) == 0) {
+            if (($flag_bits & 1) === 0) {
```

Comment:
could you explain under which circumstances this code requires the `===` instead of `==`. Same comment for the change below